post_results.py

```diff
@@ -268,6 +269,55 @@ def report_mongo(host, test_groups, elapsed_time, log):
     else:
         return None
 
+def report_slack(host, test_groups, elapsed_time, log):
+    if slackurl:
```

Comment:
use `slackurl` check to where you call this function instead of having the whole code of the function in a single block.

Reply:
done